algorithms/aeskw.js

```diff
@@ -45,16 +45,16 @@ class Kek {
   async unwrapKey({wrappedKey}) {
     const kek = this.key;
     // Note: `AES-GCM` algorithm name doesn't matter; will be exported raw.
-    wrappedKey = base64url.decode(wrappedKey);
+    wrappedKey = new Uint8Array(base64url.decode(wrappedKey));
```

Comment:
It appears to me that the return value of `base64url.decode` is already a Uint8Array.

https://github.com/digitalbazaar/base64url-universal/blob/master/lib/base64url.js#L87-L109

Reply:
You're right, I've removed it here 3dd73ee. Thanks!